### PR TITLE
New task for bitbucket

### PR DESCRIPTION
**Pull request by AsfaMumtaz**

### CODE REVIEW

File: stakater-push-main-tag-bitbucket/helm/templates/clustertask.yaml [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+        In case of PR, PR number that is to be used in image tag. If this field
+        is empty it means that it's a commit on main branch
+    - name: GIT_REVISION
+      type: string
+      description: The git revision
+    - name: BITBUCKET_TOKEN
+      type: string
+      default: "git-auth"
+      description: Secret that stores the Access Token and User Email for BitBucket
+    - name: repoPath
```
Comment (rasheedamir) [inline stakater-push-main-tag-bitbucket/helm/templates/clustertask.yaml:23]: @AsfaMumtaz plz follow same naming convention

File: stakater-push-main-tag-bitbucket/README.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
+# Stakater Cluster Task for Pushing Tags to Bitbucket Repository
```
Comment (rasheedamir) [inline stakater-push-main-tag-bitbucket/README.md:1]: @AsfaMumtaz plz add reason here that why we have created separate push tag for bitbucket

File: stakater-push-main-tag-bitbucket/helm/templates/clustertask.yaml (hunk 1 above)
Comment (rasheedamir) [reply stakater-push-main-tag-bitbucket/helm/templates/clustertask.yaml:23]: do it right now plz; we shouldn't wait

**Comment by mustafaStakater** [comment]
Github actions missing

File: .github/workflows/stakater-push-main-tag-bitbucket.yaml [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+      - main
+    paths:
+      - 'stakater-push-main-tag-bitbucket/**'
+
+  push:
+    branches:
+      - main
+    paths:
+      - 'stakater-push-main-tag-bitbucket/**'
+      - '.github/workflows/stakater-push-main-tag-bitbucket.yaml'
```
Comment (mustafaStakater) [inline .github/workflows/stakater-push-main-tag-bitbucket.yaml:14]: rename the file to use underscores instead of dashes and update this line.

**Review by mustafaStakater — APPROVED** [review]

**Pull request merged**